### PR TITLE
101 remaining issue on the mljinterface

### PR DESCRIPTION
**Pull request by Rockdeldiablo**
changed predict and mlj_flux_interface so that they work together well.

### CODE REVIEW

**Comment by Rockdeldiablo** [comment]
@MojiFarmanbar @pat-alt  it seems to work now. maybe i    made some mistake in the previous commits. Anyway i could only partially improve the predict method because predict does not accept iterators so it doesn't work directly with `collect(eachrow(Xnew))`

File: src/mlj_flux.jl [hunk 1]
```diff
@@ -448,13 +457,18 @@ function MLJFlux.predict(model::LaplaceClassification, fitresult, Xnew)
...
     #convert in a vector of vectors because Laplace ask to do so
-    X_vec = X_vec = [Xnew[i, :] for i in 1:size(Xnew, 1)]
-
-    # Predict using Laplace and collect the predictions
+    X_vec = collect(eachrow(Xnew))
     predictions = [
-        LaplaceRedux.predict(
-            la, x; link_approx=model.link_approx, predict_proba=model.predict_proba
-        ) for x in X_vec
+        map(
```
Comment (pat-alt) [inline src/mlj_flux.jl:462]: Could just try of broadcasting works here (i.e. `predict.(la, X_new)`)

Comment (pat-alt) [reply src/mlj_flux.jl:462]: Let's just open a new issue for this.

**Review by pat-alt — APPROVED** [review]
Let's just open a new issue for broadcasting `predict`

**Review by MojiFarmanbar — APPROVED** [review]

**Pull request merged**